docs/pages/management/operations/ca-rotation.mdx

```diff
@@ -1,261 +1,462 @@
+   user CA       never updated
+   db CA         never updated
+   db_client CA  never updated
+   openssh CA    never updated
+   jwt CA        never updated
+   saml_idp CA   never updated
+   oidc_idp CA   never updated
+   CA pin        sha256:0000000000000000000000000000000000000000000000000000000000000000
+   ```
+
+1. Configure and check resources depending on the CA you are rotating. This is
+   your final chance to update Teleport-protected resources before transitioning
+   to the `standby` phase.
+
+1. If you have lost connectivity to Teleport-protected resources, [roll
+   back](#rollback) to the original certificate authority before entering the
+   final `standby` phase, when rolling back is no longer possible.
+
+### Final `standby`
+
+Before wrapping up, verify that you have not lost access to Teleport-protected
+resources that rely on the CA that you rotated.
+
+1. Execute the transition:
+
+   ```code
+   $ tctl auth rotate --manual --type=<Var name="type" description="Certificate authority to rotate"/> --phase=standby
+   ```
+
+1. Verify that the rotation has completed with `tctl`:
+
+   ```code
+   $ tctl status
+   Cluster       teleport.example.com
+   Version       (=teleport.version=)
+   host CA       rotated Sep 20 2023 02:11:25 UTC
+   user CA       never updated
+   db CA         never updated
+   db_client CA  never updated
+   openssh CA    never updated
+   jwt CA        never updated
+   saml_idp CA   never updated
+   oidc_idp CA   never updated
+   CA pin        sha256:0000000000000000000000000000000000000000000000000000000000000000
+   ```
+
+1. Follow the instructions for your CA to ensure that you can connect to
+   Teleport-protected resources that rely on the CA that you rotated. This is
+   the last stage where you have the opportunity to roll back. If you have lost
+   connectivity to Teleport-protected resources, [roll back](#rollback) to the
+   original certificate authority.
+
+## Step 3/4. [Optional] Run a semi-automatic rotation
+
+You can instruct Teleport to manage the CA rotation semi-automatically.
+Semi-automatic rotation transitions between the phases of a rotation for you,
+and there is no need to run a `tctl auth rotate` command each phase. After a
+**grace period** elapses, the Teleport Auth Service updates the phase of the CA
+rotation to the next step. 
+
+### Determine whether to use a semi-automatic rotation
+
+Aside from automatic phase updates, a semi-automatic rotation is identical to a
+manual one. It is up to the operator to update any infrastructure to accommodate
+the current phase before the grace period elapses.
+
+Teleport does not check the status of any infrastructure that relies on the CA,
+meaning that you can lose connectivity if things go wrong. As a result, you
+**should not** carry out a semi-automatic rotation if you need to export a CA to
+your infrastructure.
```

Comment:
@zmb3 what do you think about a code change to just reject automatic rotation requests for several of our CAs? For example, I can't think of a legit use case for doing automatic rotation with `db` or `db_client` CAs, because you have to reissue all of your database certificates or reconfigure them to trust the new CA cert. An automatic rotation will just break access to self-hosted databases like a ticking time bomb.

As far as I know, host and user CA automatic rotation is fine (maybe not user if there are desktops configured?). Not sure about other ones.